### PR TITLE
Enable Default K3s Ingress in Liqoctl

### PR DESCRIPTION
**Pull request by aleoli**
# Description

This pr add the possibility to create an ingress resource with the `liqoctl` install command on providers:

* `kubeadm`
* `k3s` (enabled by default)

# How Has This Been Tested?

- [x] locally
- [x] on k3s installation

### CODE REVIEW

**Review by palexster — APPROVED** [review]

File: deployments/liqo/templates/liqo-auth-service.yaml [hunk 1]
```diff
@@ -18,7 +18,11 @@ spec:
     {{- include "liqo.selectorLabels" $authConfig | nindent 4 }}
     {{- include "liqo.authServiceLabels" . | nindent 4 }}
   ports:
+    {{- if not .Values.auth.tls }}
     - name: https
+    {{- else }}
+    - name: http
+    {{- end }}
```
Comment (giorio94) [inline deployments/liqo/templates/liqo-auth-service.yaml:25]: Seems reversed to me

**Review by giorio94 — APPROVED** [review]
Just a couple of thought:
1. Is it worth to offer the possibility to create the ingress through liqoctl? Would personally avoid to increase the complexity duplicating all helm parameters in the command line tool if not necessary.
2. Unclear to me why the ingress is enabled by default in k3s

**Comment by aleoli** [comment]
1. it gives you the possibility to create a basic ingress with the default class, setting all the other required (and not trivial) parameters for the service. This can be a good starting point for a user that can then template the values file created by the `--only-output-values` flag
2. The ingress is enabled by default on k3s, because k3s deploys by default a traefik ingress controller that binds the port 443 on every node. This prevents the Liqo Auth LoadBalancer Service to get an external IP.

**Pull request closed without merging**